addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -498,7 +498,7 @@ export const editorCommands = {
         // Color the selected <font>s and remove uncolored fonts.
         for (const font of new Set(fonts)) {
             colorElement(font, color, mode);
-            if (!hasColor(font, mode) && !hasColor(font, mode)) {
+            if (!hasColor(font, mode) && !font.hasAttribute('style')) {
```

Comment:
Does that property always exist?

Reply:
`fonts` is an array of fontElement.

so Yes